### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by JoscelynJames** [mui-storyblok/mui-storyblok#143]
**Is your feature request related to a problem? Please describe.**
[Current Docs](http://mui-storyblok-style-guide.com.s3-website-us-west-2.amazonaws.com/)
The current documentation is lacking context and is geared more for developers. These docs reference, props, methods, HTTP methods, and more that are not initiative non-techinicall roles.

**Describe the solution you'd like**
For consumers of this tool (marketing and other non-technical roles), we will need documentation in plain English on what each storyblok component takes in and what is required, how it looks, when and where it's available to use, and how to customize. 

**Describe alternatives you've considered**
This is documentation there is no alternative.

**Comment by apjames93** [comment]
[component-docs  work in progress!](http://mui-storyblok-component-docs.com.s3-website-us-east-1.amazonaws.com/)

We are working on more docs for users and still have alot to do. feel free to check out what we have so far.

Closing this issue.
for new issues on docs submit them at the link below
[component-docs issues](https://github.com/mui-storyblok/component-docs/issues)
